docs/_sponsortoken.mdx

```diff
@@ -1,4 +1,4 @@
 ```yaml title="evcc.yaml"
 # trial token, valid until 2024-02-08
-sponsortoken: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3MDczOTAwMDAsImlhdCI6MTcwNTc0ODQwMCwiaXNzIjoiZXZjYy5pbyIsInN1YiI6IlRyaWFsIFNwb25zb3JzaGlwIPCfppwiLCJzcGUiOnRydWUsInNyYyI6Im1hIn0.DHve76Kre6Ii78E4qc2G0E2szvcvNgFu-eF4SnxCWAc.eyJleHAiOjE3MDU2NzI4MDAsImlhdCI6MTcwNDIwNDAwMCwiaXNzIjoiZXZjYy5pbyIsInN1YiI6IlRyaWFsIFNwb25zb3JzaGlwIPCfppEiLCJzcGUiOnRydWUsInNyYyI6Im1hIn0.wqyLb0VIJ9_oM91wGyKmX3T4W4b9LPCu39DWeJQ28y4
+sponsortoken: sponsortoken: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3MDczOTAwMDAsImlhdCI6MTcwNTc0ODQwMCwiaXNzIjoiZXZjYy5pbyIsInN1YiI6IlRyaWFsIFNwb25zb3JzaGlwIPCfppwiLCJzcGUiOnRydWUsInNyYyI6Im1hIn0.DHve76Kre6Ii78E4qc2G0E2szvcvNgFu-eF4SnxCWAc
```

Comment:
Thanks for noticing! Looks like this fix also contains a copy/paste error 😂

`sponsortoken: sponsortoken: `

I've now hopefully fixed it in main.